src/Login/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Input,
+  FormControl,
+  Heading,
+  VStack,
+  useToast,
+} from 'native-base';
+import Constants from 'expo-constants';
+
+const API_URL = Constants?.manifest?.extra?.API_URL;
+
+export const LoginScreen = (props: LoginProps) => {
+  const [show, setShow] = React.useState<boolean>(false);
```

Comment:
Rather than multiplying your states, you could use a [reducer](https://fr.reactjs.org/docs/hooks-reference.html#usereducer)

Reply:
I disagree. in this case (multiple form inputs) multiple states make more sense.